### PR TITLE
New package: python-ldap-2.4.25

### PR DESCRIPTION
**Pull request by the-maldridge**

### CODE REVIEW

**Comment by Vaelatern** [comment]
No python 3 subpackage?

**Comment by the-maldridge** [comment]
Upstream currently does not support building on 3.x

File: srcpkgs/python-ldap/template [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# Template file for 'python-ldap'
+pkgname=python-ldap
+version=2.4.25
+revision=1
+build_style=python-module
+pycompile_module="ldap"
```
Comment (Vaelatern) [inline srcpkgs/python-ldap/template]: Should also have the `.py` files under `site-packages` in a space separated list.

Comment (Vaelatern) [inline srcpkgs/python-ldap/template]: Examples include the `z3` package.

**Comment by the-maldridge** [comment]
Updated per request

File: srcpkgs/python-ldap/template [hunk 2]
```diff
@@ -0,0 +1,23 @@
+# Template file for 'python-ldap'
+pkgname=python-ldap
+version=2.4.25
+revision=1
+build_style=python-module
+pycompile_module="lidf.py sasl.py logger.py register.py dds.py
+ tokenizer.py subentry.py models.py modlist.py filter.py syncrepl.py
+ ldapobject.py functions.py deref.py pwdpolicy.py openldap.py
+ libldap.py readentry.py sessiontrack.py ppolicy.py simple.py
+ psearch.py dn.py async.py cidict.py dsml.py ldapurl.py"
```
Comment (Vaelatern) [inline srcpkgs/python-ldap/template]: Apologies, should have specified .py files to a depth of 1. Thus: `ldap ldif.py ldapurl.py dsml.py`
Still not sure a shared object file should be in site-packages.

**Pull request merged**